### ISSUE TITLE
[Feature Request] Add / Remove pets from feeder

### ISSUE THREAD

**Issue by klose54** [benleb/surepetcare#64]
Hello,

Could it be possible to have the action to remove / add a pet to the feeders ?

I woud like to add a cat to a full of food feeder and remove it automatically when an automatic calcul says it's too much.

**Comment by rasmi** [comment]
I have been able to implement this in HA with https://github.com/fabieu/sureflap-api running as an add-on, and triggered by a rest_command automation. I will post details shortly.

**Comment by flemscom** [comment]
@rasmi Can you reveal a bit of what you did? How did you get fabieu's api work to run as an addon in HA?

**Comment by rasmi** [comment]
@flemscom Yes, see here: https://gist.github.com/rasmi/c67e33671fe40f073657e0f3b1c4bdc5

And then I created a set of actions(?) which query the REST API to enroll/unenroll my pet at a certain hour. HOWEVER, I gave up because there was sometimes significant lag (many minutes) between the enrollment change and the device updating, I imagine the feeder does not poll/update this field regularly (for example -- not sure!).
